src/components/SerialConsole/console.css

```diff
@@ -0,0 +1,56 @@
+.console-ct {
+    font-family: Menlo, Monaco, Consolas, monospace;
+    margin-top: 0;
+    margin-bottom: 0;
+    font-size: 10px;
+    text-align: center;
+    line-height: normal;
+}
+
+@media (min-width: 568px) {
+    .console-ct {
+        font-size: 12px;
+    }
+}
+
+.console-ct > pre {
+    padding: 10px;
+    text-align: left;
+    display: block;
+    font-family: inherit;
+    font-size: inherit;
+    width: 48em;
+    height: 310px;
+    overflow-y: scroll;
+    white-space: pre-wrap;
+    margin: 0 auto;
+}
+
+.console-ct > .terminal {
+    color: #F0F0F0;
+    text-align: left;
+    outline: medium none;
+    background-color: black;
+    border: 1px solid black;
+    padding: 10px;
+}
+
+.terminal .terminal-cursor {
+    border: 1px solid #f0f0f0;
+}
+
+.terminal:focus .terminal-cursor {
+    border: none;
+    animation: blink 1s step-end infinite;
+}
+
+@keyframes blink {
+    from {
+        color: #000;
+        background: #f0f0f0;
+    }
+    50% {
+        color: #f0f0f0;
+        background: #000;
+    }
+}
```

Comment:
This should be done in less and sass and added to the appropriate directories (we will need to add the converter to this repo soon).
Colors should be from the patternfly palette.

Reply:
ok, css removed.